### PR TITLE
feat: Added support for django3.2

### PR DESCRIPTION
**Pull request by Jawayria**
JIRA: https://openedx.atlassian.net/browse/BOM-2624

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -10,34 +10,35 @@ ignore = D101,D200,D203,D212
...
 deps = 
-	-r{toxinidir}/requirements/quality.txt
-	-r{toxinidir}/requirements/test.txt
+    -r{toxinidir}/requirements/quality.txt
+    -r{toxinidir}/requirements/test.txt
 commands = 
-	touch tests/__init__.py
-	pylint help_tokens tests
-	rm tests/__init__.py
-	pycodestyle help_tokens tests
```
Comment (mraarif) [inline tox.ini:38]: we should undo these changes

File: tox.ini [hunk 2]
```diff
@@ -10,34 +10,35 @@ ignore = D101,D200,D203,D212
 match-dir = (?!migrations)
 
 [pytest]
-DJANGO_SETTINGS_MODULE = test_settings
+django_settings_module = test_settings
```
Comment (mraarif) [inline tox.ini:13]: this change should be reverted

**Review by mraarif — APPROVED** [review]

**Review by awais786 — APPROVED** [review]
